Drivers/Inc/Switches.h

```diff
@@ -0,0 +1,27 @@
+#ifndef SWITCHES_H
+#define SWITCHES_H
+#include "BSP_SPI.h"
+#include "BSP_GPIO.h"
+#include "common.h"
+#include "config.h"
+
+
+typedef enum {CRUZ_st=0, CRUZ_EN, HZD_SQ, FR_SW, Headlight_SW, LEFT_SW, RIGHT_SW, REGEN_SW, IGN_1, IGN_2, REV_SW} switches_t;
+
+/**
+ * @brief   Initializes all switches
+ *          from the steering wheel
+ * @param   None
+ * @return  None
+ */ 
+void Switches_Init(void);
+
+/**
+ * @brief   Reads the current state of 
+ *          the specified switch
+ * @param   sw the switch to read
+ * @return  State of the switch (ON/OFF)
+ */ 
+State Switches_Read(switches_t sw);
```

Comment:
Is this really the right return value for this? Some of our switches are three-state and the `State` type only has `ON` and `OFF`.

Reply:
I guess the real question is a which layer we need to worry about the difference between two switches and a three-state switch, if ever.

Reply:
We have 2 values in the enum for the 3-state switches. So we have a `RIGHT_SW` and `LEFT_SW` to handle for that.